PIE/ROV_SLAVE_SKELETON_zadian.ino

```diff
@@ -1,146 +0,0 @@
-#include <Wire.h>  // i2c library
```

Comment:
damnit

Reply:
I Made these changes so accept this PR FINALLY.